src/lxc/monitor.c

```diff
@@ -181,7 +181,6 @@ int lxc_monitor_sock_name(const char *lxcpath, struct sockaddr_un *addr) {
 	if (ret < 0)
 		return -1;
 
-	sockname[sizeof(addr->sun_path)-3] = '\0';
```

Comment:
The `-3` looks suspicious btw. It used to be `-2` which makes more sense, should check why that errored. Taking this away however is definitely a problem due to the address being checked with `strlen()` and `snprintf` potentially truncating the data (`ret == len`).